### ISSUE TITLE
Option to set ALL folders to a desired sorting (last edited, A-Z, ...)

### ISSUE THREAD

**Issue by Rowin63** [osmandapp/OsmAnd#21060]
After a fresh install and bring back all my folders (50+) into osmand, ALL folders are set to "last edited" again, and this sorting is really annoying. So now I have to go through every single folder and set it to "A-Z" again.

A possibility to set ALL folders to a certain sorting option would be great!
Also when creating or renaming a folder its standard sorting is always this annoying "last edited".
Let the user set a standard setting please, thanks!

**Comment by vshcherb** [comment]
There is an option to change recursively probably you don't need 50 top level track folders?

**Comment by Rowin63** [comment]
> There is an option to change recursively probably you don't need 50 top level track folders?

I see, there is an option "Unterordner sortieren" ("sorting subfolders") in my places´ 3dots. 
I did not expect this option sorting the _tracks_, if it is named "sorting _subfolders_"   ;-)

My tracks are sorted by country_year, source (different sources per country) and after travelling already for some years, 50 folders is not much.

But you´re right, it´s a lot of folders and this was my intention for creating various sub-folders inside a country folder.
What unfortunately does not really work properly, as we know by now ;-)

**Comment by sonora** [comment]
I have decided to put all my own tracks in _one single top folder_, which makes things like this easier...

**Comment by Rowin63** [comment]
Yes, maybe, but I don't want to leave my used "folder creating system" because "last edited" is the standard sorting type...

**Comment by sonora** [comment]
> Yes, maybe, but I don't want to leave my used "folder creating system" because "last edited" is the standard sorting type...

In fact, for those of us having big libraries of tracks, A-Z is usually the method of choice for sorting them, becasue we would likely automatically use a file name structure fulfilling exactly our needs of structuring and locating them. "Last edited" is probably a comparable small use case, only valid for
  - **recordings**. If at all, our standard naming starts all recording with an ISO date so A-Z sorting is equally useful there.
  - and users creating or modifying tracks by **edits**. But this may often only happen in certain folders, not across the board.

So in summary, we could maybe derive a suggestion that
- "Last edited" should at most only be the default sorting in some folders, perhaps in the system folders "rec" and "import",
- while "A-Z" should be the default sorting for all user folders / custom folders?

**Comment by Rowin63** [comment]
Thats exactly how I use it.
"rec" and "import" are the only folders set to "recently updated". So I can see the latest/newest tracks.
Thanks for your opinion!
